examples/Folders/Readme.md

```diff
@@ -0,0 +1,49 @@
+Dropzone that accepts folders as drag-and-drop.
+
+```
```

Comment:
This is missing language. Please add `jsx harmony` here. This will enable syntax highlighting on GitHub and code editors.

Reply:
@okonet  I am not sure about this one. Shall I just add `// language=JSX Harmony` at the top of this file? It is the only thing I found for setting language explicitly, is it right?

Reply:
no it would just need to be added like so ` ```jsx harmony`

Reply:
@rkmarks Thanks!